lab3/src/cv_module.py

```diff
@@ -145,40 +89,46 @@ def area_check(self, rgb_img):
         # RED_TOL = [10, 50, 20]  # Red hue tolerance
 
         self.hsv_img = cv2.cvtColor(rgb_img, cv2.COLOR_BGR2HSV)
-        self.mask_red = self.hsv_thresh(red_samp, RED_TOL)
```

Comment:
Leave this in, but uncomment the RED_TOL and BLUE_TOL definitions in line 144 and 145.

Reply:
Alrighty